cli/config/cache.php

```diff
@@ -1,8 +1,19 @@
 <?php
+declare(strict_types=1);
```

Comment:
I'd very much prefer all the configuration options be in one file - the `config.php` one. Also the adapter might not work here because the config (as shown below) applies only to libmemcached